cmd/api/discovery/apiresources.go

```diff
@@ -53,3 +47,17 @@ func GetAPIResource(client rest.Interface) gin.HandlerFunc {
 			http.StatusBadRequest)
 	}
 }
+
+func getDiscoveryURL(group, version string) string {
+	groupVersion := version // Base case: core resource, just version
+	if group != "" {
+		groupVersion = fmt.Sprintf("%s/%s", group, version)
+	}
+
+	url := fmt.Sprintf("/api/%s", groupVersion) // Base case: core resource, /api as root
+	if group != "" {
+		url = fmt.Sprintf("/apis/%s", groupVersion)
+	}
```

Comment:
You have duplicate conditionals here - you should consolidate the logic and remove the intermediate `groupVersion`:

```suggestion
	url := fmt.Sprintf("/api/%s", version) // Base case: core resource, just version, /api as root
	
	if group != "" {
		url = fmt.Sprintf("/apis/%s/%s", group, version) // Extended case: group and version, /apis as root
	}
```

Reply:
Good catch Chris! This is much simpler.